src/Datadog.Trace/CorrelationIdentifier.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+
+namespace Datadog.Trace
+{
+    /// <summary>
+    /// An API to access the active trace and span ids.
+    /// </summary>
+    public static class CorrelationIdentifier
+    {
+        // private static readonly string TraceIdKey = "dd.trace_id";
```

Comment:
Currently commented out because they are unused (causing a compiler error), but they will be used when implementing logging integrations.